r/R/atac_utils.R

```diff
@@ -593,6 +603,8 @@ call_macs_peaks <- function(fragments, cell_groups, path, effective_genome_size
                                                       "pValue", "qValue", "pointSource"),
                                           show_col_types = FALSE)
       peaks[[cluster]]$cluster <- cluster
```

Comment:
Could we actually name this column "group"? I'm not 100% sure on this, but I'm trying to think of consistency with the `cell_groups` argument name and not assuming that the cell groups correspond directly to clusters. 

I should have mentioned this on the previous review round, sorry